app/domain/data.ts

```diff
@@ -19,6 +19,12 @@ export type GeoShape = {
   geometry: GeoJSONGeometry;
 };
 
+export type GeoPoint = {
+  iri: string;
+  label: string;
+  coordinates: [number, number];
```

Comment:
Nit: better to do `{ iri:string, label:string, longitude:number, latitude:number }`?

Reply:
Ah, yes! Anyway I think this will be needed to change when merging with your branch :)